indico/modules/events/management/templates/contribution_person_list.html

```diff
@@ -28,7 +28,7 @@
             </div>
         </div>
     </div>
-    <div class="form-group">
+    <div class="form-group table-content">
```

Comment:
Also, the scroll bars shouldn't be visible when it's not necessary. Try `overflow: auto`